src/Microsoft.Health.Dicom.Api/Web/DicomStreamContent.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.Health.Dicom.Api.Web;
 
-public class DicomStreamContent
+public class DicomStreamContent : IHttpStreamContent
```

Comment:
Will need an engineering review for these files. :-)

Reply:
Yes, I will ask Will to take a quick look